performance-timeline/case-sensitivity.any.js

```diff
@@ -27,3 +27,37 @@
     assert_equals(mixedList.length, 0, "getEntriesByName('" + location2 + "').length");
 
   }, "getEntriesByName values are case sensitive");
+
+  async_test(function (t) {
+    // Test type/buffered case sensitivity.
+    observer = new PerformanceObserver(
+      t.step_func(function (entryList, obs) {
+        assert_unreached("Observer should not be called.");
+      })
+    );
+    observer.observe({type: "Mark"});
+    observer.observe({type: "Measure"});
+    observer.observe({type: "MARK"});
+    observer.observe({type: "MEASURE"});
+    observer.observe({type: "Mark", buffered: true});
+    observer.observe({type: "Measure", buffered: true});
+    observer.observe({type: "MARK", buffered: true});
+    observer.observe({type: "MEASURE", buffered: true});
+    self.performance.mark("mark1");
+    self.performance.measure("measure1");
+    t.step_timeout(function() {
```

Comment:
nit: let's move this to the end of the function so that it starts waiting only when everything else in the test has been executed.